src/parser_utils.c

```diff
@@ -0,0 +1,112 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   parser_utils.c                                     :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: sqiu <sqiu@student.42vienna.com>           +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/07/22 13:15:09 by sqiu              #+#    #+#             */
+/*   Updated: 2023/07/29 13:37:30 by sqiu             ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+/**
+ * @file parser_utils.c
+ * @brief Helper functions for parser.
+ */
+
+#include "mod_parser.h"
+
+/**
+ * @brief Iterate to last node of given list.
+ * 
+ * Returns 0 if list is empty.
+ * @param cmd_head 		List of cmd-structures
+ * @return t_cmd* 
+ */
+t_cmd	*ft_last_cmd(t_cmd *cmd_head)
```

Comment:
Maybe add "find" to function name: ft_find_last_cmd()

Reply:
Thank you for sharing ;D

No, but seriously, don't think that's necessary. It would just make it too verbose and the intention of the function should be clear enough

Reply:
Ha! I think we diverge on what is too verbose. But I don't think a change is necessary.